catroid/src/androidTest/java/org/catrobat/catroid/test/web/AuthenticationCallsTest.java

```diff
@@ -68,7 +68,7 @@
 	private static final int STATUS_CODE_USER_ADD_EMAIL_EXISTS = 757;
 	private static final int STATUS_CODE_USER_EMAIL_INVALID = 765;
 	private static final int STATUS_CODE_AUTHENTICATION_FAILED = 601;
-	private static final int STATUS_CODE_USER_NOT_EXISTING = 764;
+	private static final int STATUS_CODE_USER_NOT_EXISTING = 803;
```

Comment:
what is this doing in this PR?

Reply:
the share guys changed the status code of user non existing to 803 and because these the loginWithNonExisitingUser failed I wanted to fix it as well within this PR. I know it should be a separated ticket but it was a overkill in my eyes

Reply:
the share guys changed their status code from 764 to 803. Therefore the testcase was failing all the time. To see a whole pass on jenkins changed it to 803. And to reduce opening a new PR i changed it in these PR.